### PR TITLE
Add PubSub Tutorial for Cloud Run

### PR DESCRIPTION
**Pull request by averikitsch**

### CODE REVIEW

File: appengine-java11/cloudrun-pubsub/pom.xml [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+          <executions>
+              <execution>
+                  <goals>
+                      <goal>repackage</goal>
+                  </goals>
+              </execution>
+          </executions>
+      </plugin>
+
+      <plugin>
```
Comment (lesv) [inline appengine-java11/cloudrun-pubsub/pom.xml]: Why is this here?

File: appengine-java11/cloudrun-pubsub/src/main/java/com/example/appengine/cloudrunpubsub/Body.java [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.appengine.cloudrunpubsub;
+
+// Body.Message is the payload of a Pub/Sub event. Please refer to the docs for
+// additional information regarding Pub/Sub events.
+public class Body {
```
Comment (lesv) [inline appengine-java11/cloudrun-pubsub/src/main/java/com/example/appengine/cloudrunpubsub/Body.java]: Usually a good idea to implement hash, compare, etc.  Not important for this, but in general it's a good practice.

File: appengine-java11/cloudrun-pubsub/pom.xml (hunk 1 above)
Comment (averikitsch) [reply appengine-java11/cloudrun-pubsub/pom.xml]: Removed.

File: appengine-java11/cloudrun-pubsub/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+COPY src ./src
+
+# Build a release artifact.
+RUN mvn package -DskipTests
+
+# Use AdoptOpenJDK for base image.
+# It's important to use OpenJDK 8u191 or above that has container support enabled.
+# https://hub.docker.com/r/adoptopenjdk/openjdk11
+# https://docs.docker.com/develop/develop-images/multistage-build/#use-multi-stage-builds
+FROM adoptopenjdk/openjdk11:x86_64-alpine-jdk-11.0.3_7-slim
```
[the remark this answers is not in the document]
Comment (averikitsch) [reply appengine-java11/cloudrun-pubsub/Dockerfile]: This is for a small image size.
